yarn-project/noir-protocol-circuits/src/type_conversion.ts

```diff
@@ -87,7 +87,7 @@ import {
   AztecAddress as NoirAztecAddress,
   EthAddress as NoirEthAddress,
   Field as NoirField,
-  Point as NoirPoint,
+  GrumpkinPoint as NoirPoint,
```

Comment:
Why does it suddenly become `GrumpkinPoint`?

Reply:
there was a small type change in https://github.com/AztecProtocol/aztec-packages/pull/3814/files